packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -168,11 +209,19 @@ export class HbarLimitService implements IHbarLimitService {
       }`,
     );
 
+    // Check if the spending plan is being used for the first time today
+    if (spendingPlan.spentToday === 0) {
+      this.dailyUniqueSpendingPlansCounter[spendingPlan.subscriptionType].inc(1);
+    }
+
     await this.hbarSpendingPlanRepository.addAmountToSpentToday(spendingPlan.id, cost);
     await this.hbarSpendingPlanRepository.addAmountToSpendingHistory(spendingPlan.id, cost);
     this.remainingBudget -= cost;
     this.hbarLimitRemainingGauge.set(this.remainingBudget);
 
+    // Done asynchronously in the background
+    this.updateAverageDailyUsagePerSubscriptionType(spendingPlan.subscriptionType).then();
```

Comment:
Yes, it adds explicitness that this call is not "awaited". (otherwise we get a typescript warning to add `await` to the invocation of the async method)